versions.json

```
@@ -51,8 +51,8 @@
       "buster/apache",
       "buster/fpm",
       "buster/zts",
-      "alpine3.12/cli",
-      "alpine3.12/fpm"
+      "alpine3.13/cli",
+      "alpine3.13/fpm"
```

In this case, `8.0` is now a released version (not prerelease), so we should keep `alpine3.12`:

```suggestion
      "alpine3.13/cli",
      "alpine3.13/fpm",
      "alpine3.12/cli",
      "alpine3.12/fpm"
```

(And the associated folder footwork to make that work -- I'd recommend just copying `alpine3.13` over to `alpine3.12` and then running the templating scripts again.)

---

Thanks for the feedback @tianon, I reverted the removal of Alpine 3.12 for PHP 8.0 which makes it now available for images building